docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,326 @@
+
+Change the value of `EnforcePassword` to `0` (default is `1`).
+
+Extend URL length for the request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Best practices from Cisco recommand to extend the URL length to avoid issue with longer request. 
+
+To do so, enter the following command in the CLI on the NDES server:
+
+ %systemroot%\system32\inetsrv\appcmd.exe set config /section:system.webServer/
+ security/requestFiltering /requestLimits.maxQueryString:"8192" /commit:apphost
+
+Certificate template
+^^^^^^^^^^^^^^^^^^^^
+
+NOTE: Remember that the validity of your CA can impact your whole certificate architecture.
+
+The settings to change need to be access via `Server Manager` > `Roles` > `Active Directory Certificates Services` > `Certificate templates`.
+
+The main goal is to deliver certificate for `user Authentication`, this mean we have to setup a specific template.
+
+First, the certificate template need at least the following `Enhanced Key Usage`:
+
+* Client Authentication
+
+and the following `Key Usage`:
+
+* Digital signature
+* Key encipherment
+
+Since we will duplicate a template, those `Key Usage`, and `Enhanced Key Usage` are already configured. We advise to duplicate the template `User` and change the necessary settings.
+
+Once duplicated, right click your new template, go to `Properties`. Navigate to the tab `Subject Name`. Make sure to select `Supplied in the request` over `Built from information in Active Directory`, else the requested CN will be overwritten by NDES.
+
+Now that you chose your template to deliver you need to allow the use of this template by NDES and configure it in the registry.
+
+To allow NDES to use this template you need to navigate to `Server Manager` > `Roles` > `Active Directory Certificates Services` > expand `YourServerName`, right click `Certificate template` and chose `New template to issue`, in the list select your newly created template.
+
+For the registry, press `Start` and type `regedit` in the search bar.
+
+While in the registry navigate to `Computer` > `HKEY_LOCAL_MACHINE` > `SOFTWARE` > `Microsoft` > `Cryptography` > `MSCEP`.
+
+Here you should have a list of three keys entries: 
+
+* EncryptionTemplate, 
+* GeneralPurposeTemplate,
+* SignatureTemplate.
+
+Replace each value with our newly created template. The default value should be `IPSECIntermediateOffline`.
+
+You now need to reboot the NDES server to apply changes to the registry.
+
+IIS configuration
+^^^^^^^^^^^^^^^^^
+
+The use of SCEP with PacketFence also require a change in the IIS configuration.
+
+Navigate to `Server Manager` > `Web(IIS)` > expand `Default web site` > select `CertSrv` > `mscep`. Select `Authentication`, and double click `Anonymous Authentication`. Make sure that `Application pool identity` is selected.
+
+Online Certificate Status Protocol (OCSP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the configuration of OCSP with a Microsoft Windows 2008 R2, there are small changes to apply.
+
```

Comment:
Before trying to create a OCSP Responder we need to include a new template in the CA (OCSP Response Signing)